packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -88,17 +90,36 @@ export default function ReusableBlockConvertButton( {
 	const onConvert = useCallback(
 		async function ( reusableBlockTitle ) {
 			try {
-				await convertBlocksToReusable( clientIds, reusableBlockTitle );
-				createSuccessNotice( __( 'Reusable block created.' ), {
-					type: 'snackbar',
-				} );
+				await convertBlocksToReusable(
+					clientIds,
+					reusableBlockTitle,
+					syncType
+				);
+				createSuccessNotice(
+					sprintf(
+						// translators: %s: The sync status of the block that is created.
+						__( '%s created.' ),
+						syncType === 'fully'
+							? __( 'Synced Pattern' )
+							: __( 'Unsynced Pattern' )
```

Comment:
This is interpolating one translated string into another, and that can cause some oddities in some languages where ordering is important.

I think it'd be better to instead define two separate pieces of translated text:
```js
syncType === 'fully' ? __( 'Synced Pattern created' ) : __( 'Unsynced Pattern created' );
```

Reply:
Fixed.